src/main/java/org/taf/pages/LoginPage.java

```diff
@@ -0,0 +1,60 @@
+package org.taf.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+
+    public static final String USERNAME_LOCATOR = "input[type=\"text\"]";
+    public static final String PASSWORD_LOCATOR = "input[type=\"password\"]";
+    public static final String SUBMIT_LOCATOR = "button[type=\"submit\"]";
```

Comment:
All this constants could be private.

Reply:
Yes, they could. They were updated. Thanks